### ISSUE TITLE
Xcode 13 - waitForView(withAccessibilityIdentifier) issue

### ISSUE THREAD

**Issue by lolindrath** [kif-framework/KIF#1240]
I have a project where I test that a UILabel within a UITableView section header is set correctly. This test runs fine in Xcode 12 / iOS 14 but in Xcode 13 / iOS 15 it crashes because `waitForView` returns the `UITableView` instead of the expected `UILabel`. Here's the error:

```
Could not cast value of type 'UITableView' (0x11637a978) to 'UILabel' (0x1163852d8).
```

Steps to Reproduce:

1. Open sample project below in Xcode 12 and run the tests on a iOS 14
2. Note that the test runs fine
3. Open sample project below in Xcode 13 and run the tests on iOS 15
4. The `testLayout()` test should crash and throw the above message.

https://github.com/lolindrath/TableSectionHeaderKIF

Let me know if I can supply any more details, thanks!

**Comment by lolindrath** [comment]
Thanks @dostrander ! You're awesome!

**Comment by dostrander** [comment]
This has been released with 3.8.4
